styles/style.css

```diff
@@ -0,0 +1,69 @@
+body {
+  font-family: 'Roboto', sans-serif;
+}
+
+.calculator {
+  border: 1px solid #999999;
+  border-radius: 6px;
+  box-shadow: 0px 15px 20px 5px rgba(136, 136, 136, 0.5);
+  height: 318px;
+  margin: 200px 0px 0px 500px;
```

Comment:
this is not an effective way to center/center something on the page. Look into flexbox as well as the `margin: 0 auto` trick.